jina/clients/python/request.py

```diff
@@ -31,8 +31,8 @@ def _build_doc_from_content():
             return data, DataInputType.DOCUMENT
         try:
             d = Document(data, **kwargs)
-            if override_doc_id:
-                d.update_id()
+            #if override_doc_id:
```

Comment:
here if it may be good to provide some logic to assign random documents if client did not take care of it?

Reply:
If a `Document` is initiated in the first place, you either provide an `id` or a random one is generated.